website/docs/concepts/html/introduction.mdx

```diff
@@ -151,7 +151,41 @@ HTML code.
 At the moment the lints are mostly accessibility-related. If you have ideas for lints, please feel
 free to [chime in on this issue](https://github.com/yewstack/yew/issues/1334).
 
-## Special properties
+## Specifying attributes and properties
+
+Attributes are set on elements in the same way as in normal HTML:
+
+```rust
+let value = "something";
+html! { <div attribute={value} />
+```
+
+Properties are specified with `~` before the element name:
+
+```rust
+html! { <my-element ~property="abc" /> }
+```
+
+:::tip
+
+The braces around the value can be ommited if the value is a literal.
+
+:::
+
+:::note What classifies as a literal
+
+Literals are all valid [literal expressions](https://doc.rust-lang.org/reference/expressions/literal-expr.html)
+in Rust. Note that [negative numbers are **not** literals](https://users.rust-lang.org/t/why-are-negative-value-literals-expressions/43333)
+and thus must be encosed in curly-braces `{-6}`
+
+:::
+
+::: note Component properties
```

Comment:
```suggestion
:::note Component properties
```
This note section wouldn't show properly if a space is included.

<img width="915" alt="image" src="https://user-images.githubusercontent.com/11693215/192171131-51b0591f-cb21-474a-ae14-5791f51e1150.png">